src/studio/src/designer/DataModeling.Tests/_TestData/Model/JsonSchema/SeresNillable.json

```diff
@@ -0,0 +1,137 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "schema.json",
+  "type": "object",
+  "@xsdNamespaces": {
+    "xsd": "http://www.w3.org/2001/XMLSchema",
+    "xsi": "http://www.w3.org/2001/XMLSchema-instance",
+    "seres": "http://seres.no/xsd/forvaltningsdata"
+  },
+  "@xsdSchemaAttributes": {
+    "AttributeFormDefault": "Unqualified",
+    "ElementFormDefault": "Qualified",
+    "BlockDefault": "None",
+    "FinalDefault": "None"
+  },
+  "info": {
+    "XSLT-skriptnavn": "SERES_XSD_GEN",
+    "XSD-generatorversjon": "2.0.17",
+    "XSLT-prosessor": "SAXON versjon 9.1.0.7",
+    "generert": "2016-12-14T13:28:58.147+01:00",
+    "navneromprefix": "http://seres.no/xsd",
+    "namespace": "http://seres.no/xsd/altinn/test/basic",
+    "meldingsnavn": "melding",
+    "domenenavn": "altinn-test",
+    "modellnavn": "main",
+    "metamodellversjon": "1.2",
+    "guid": "true",
+    "orid": "false",
+    "nillable": "true",
+    "tillat-gjenbruk": "true",
+    "elementtype": "true",
+    "forvaltningsdata": "true",
+    "forvaltningsdata-navnerom": "http://seres.no/xsd/forvaltningsdata",
+    "saernorske-bokstaver-i-navn": "false",
+    "ft_guid_som_attributt": "false",
+    "sem-ref": "false",
+    "kodebibliotek": "false",
+    "spraak": "",
+    "XSD-variant": "Altinn 1.3.1"
+  },
+  "oneOf": [
+    {
+      "$ref": "#/$defs/main"
+    }
+  ],
+  "$defs": {
+    "main": {
+      "allOf": [
+        {
+          "properties": {
+            "p1": { // compatible types: SimpleType, Nillable
+              "type": [ "string", "null" ],
+              "@xsdType": "string"
+            },
+            "p2": { // compatible types: SimpleType, Nillable
+              "oneOf": [
+                {
+                  "$ref": "#/$defs/simpleType"
+                },
+                {
+                  "type": "null"
+                }
+              ]
+            },
+            "p3": { // compatible types: ComplexType, Nillable
+              "oneOf": [
+                {
+                  "$ref": "#/$defs/complexType"
+                },
+                {
+                  "type": "null"
+                }
+              ]
+            },
+            // inline nillable complex type (not sure we need to support this for SERES)
+            "p4": {
+              "oneof": [
+                {
+                  "properties": {
+                    "fish": {
+                      "type": "string",
+                      "@xsdType": "string"
+                    },
+                    "stew": {
+                      "type": "string",
+                      "@xsdType": "string"
+                    }
+                  }
+                },
+                {
+                  "type": "null"
+                }
+              ]
+            }
+          },
+          "required": [ "p1", "p2", "p3", "p4" ]
+        },
+        {
+          "properties": {
+            "p1_a": { // compatible types: SimpleType, Nillable
```

Comment:
I'm unable to visually match this to anything in the SeresNilable.xsd file.

Reply:
Yes, it's just a copy/paste example for now and not used. We identified a few more nillable cases last night, I'm working on implementing the last now and will clean up this file as part of it.

Reply:
Fixed